perfmetrics/scripts/continuous_test/gcp_ubuntu/periodic_experiments/build.sh

```diff
@@ -0,0 +1,87 @@
+  $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+sudo apt-get update
+sudo apt-get install jq
+sudo apt-get install docker-ce docker-ce-cli containerd.io docker-compose-plugin -y
+
+cd "${KOKORO_ARTIFACTS_DIR}/github/gcsfuse"
+# Get the latest commitId of yesterday in the log file. Build gcsfuse and run
+# integration tests using code upto that commit.
+commitId=$(git log --before='yesterday 23:59:59' --max-count=1 --pretty=%H)
+git checkout $commitId
+
+echo "Executing integration tests"
+GODEBUG=asyncpreemptoff=1 go test ./tools/integration_tests/... -p 1 --integrationTest -v --testbucket=gcsfuse-integration-test
+
+# Checkout back to master branch to use latest CI test scripts in master.
+git checkout master
+
+echo "Building and installing gcsfuse"
+# Build the gcsfuse package using the same commands used during release.
+GCSFUSE_VERSION=0.0.0
+sudo docker build ./tools/package_gcsfuse_docker/ -t gcsfuse:$commitId --build-arg GCSFUSE_VERSION=$GCSFUSE_VERSION --build-arg BRANCH_NAME=$commitId
+sudo docker run -v $HOME/release:/release gcsfuse:$commitId cp -r /packages /release/
+sudo dpkg -i $HOME/release/packages/gcsfuse_${GCSFUSE_VERSION}_amd64.deb
+
+cd "./perfmetrics/scripts/"
+
+echo Installing requirements..
+pip install --require-hashes -r bigquery/requirements.txt --user
+
+# Get the current date and time
+current_date=$(date +"%Y-%m-%d %H:%M:%S")
+
+# Get the required enabled configuration
+config=$(jq --arg EXPERIMENT_NUMBER "$EXPERIMENT_NUMBER" --arg current_date "$current_date" '.experiment_configuration[] | select(.end_date >= $current_date)' continuous_test/gcp_ubuntu/periodic_experiments/experiments_configuration.json | jq -s ".[$EXPERIMENT_NUMBER-1]")
+
+# Check if any enabled configurations were found
+if [ "$config" = "null" ]; then
+  echo "No enabled configuration found for value: $EXPERIMENT_NUMBER"
+  exit 1
+fi
+
+# Access specific properties of the configuration
+CONFIG_NAME=$(echo "$config" | jq -r '.config_name')
+GCSFUSE_FLAGS=$(echo "$config" | jq -r '.gcsfuse_flags')
+BRANCH=$(echo "$config" | jq -r '.branch')
+END_DATE=$(echo "$config" | jq -r '.end_date')
+
+CONFIG_ID=$(eval "python3 bigquery/get_experiments_config.py --gcsfuse_flags '$GCSFUSE_FLAGS' --branch '$BRANCH' --end_date '$END_DATE' --config_name '$CONFIG_NAME'")
+START_TIME_BUILD=$(date +%s)
+
+UPLOAD_FLAGS=""
+if [ "${KOKORO_JOB_TYPE}" == "RELEASE" ] || [ "${KOKORO_JOB_TYPE}" == "CONTINUOUS_INTEGRATION" ] || [ "${KOKORO_JOB_TYPE}" == "PRESUBMIT_GITHUB" ];
+then
+  UPLOAD_FLAGS="--upload_gs --upload_bq --config_id $CONFIG_ID --start_time_build $START_TIME_BUILD"
+fi
+
+# Executing perf tests
+LOG_FILE_FIO_TESTS="${KOKORO_ARTIFACTS_DIR}/gcsfuse-logs${EXPERIMENT_NUMBER}.txt"
+GCSFUSE_FIO_FLAGS="$GCSFUSE_FLAGS --log-file $LOG_FILE_FIO_TESTS --log-format \"text\""
+chmod +x run_load_test_and_fetch_metrics.sh
+./run_load_test_and_fetch_metrics.sh "$GCSFUSE_FIO_FLAGS" "$UPLOAD_FLAGS"
```

Comment:
We will have to use 4 different buckets for running 4 experiments at the same time because the fio write test involves writing to buckets and if all 4 experiments are writing to same bucket then it can cause inconsistency. I would suggest you pass another flag to run_load_test_and_fetch_metrics & run_ls_benchmark for bucket name. If bucket name is not passed then those scripts can use the default bucket which is currently used in existing setup.